### ISSUE TITLE
Mark System Apps & Disabled Apps

### ISSUE THREAD

**Issue by NikunjKhangwal** [butzist/ActivityLauncher#253]
Exactly the title. Just a little qol suggestion.

**Comment by butzist** [comment]
Good idea, but unless we get a PR, not likely before PlayStore release

**Comment by Sash0k** [comment]
@NikunjKhangwal fyi, implemented in fork

**Comment by NikunjKhangwal** [comment]
> @NikunjKhangwal fyi, implemented in fork

Yeah that's where i got the idea from
